run.sh

```diff
@@ -1,13 +1,42 @@
 #!/bin/bash
 export MIN_JS=false
 
-ADD=""
+PROFILE="dev"
+CREATE_DB=""
+YARN="-Dskip.yarn"
```

Comment:
@joshsadam  might have opinions here, but I feel like the default should be to *not* skip yarn.  I feel like it's pretty regular that a UI change could have been made, so it should run by default.  Plus on the first run after cloning the repo I think that yarn *must* run so we don't want to make it a non-default.

Reply:
I think by default it is fine to ignore because it is so much faster.  Just make sure that it is noted somewhere that you need to run `yarn install` and one of `yarn build` or `yarn start`